### ISSUE TITLE
!important without whitespace considered invalid in some cases

### ISSUE THREAD

**Issue by ArgonAlex** [csstree/csstree#155]
It seems like `!important` immediately preceded by a `)` is being considered an invalid value. e.g:
```css
width:calc(100% - 10px)!important;
```
```css
color:rgb(0,0,0)!important;
```

**Comment by lahmatiy** [comment]
Considered by whom? I know nothing about issues with this.

**Comment by ArgonAlex** [comment]
By csstree. I think it's a bug.
```js
console.log(require('css-tree').lexer.matchProperty('width', 'calc(33.33333% - 10px)!important;').error);
```
```
SyntaxError [SyntaxMatchError]: Mismatch
    at matchSyntax (node_modules\css-tree\lib\lexer\Lexer.js:87:17)
    at Lexer.matchProperty (node_modules\css-tree\lib\lexer\Lexer.js:331:16)
  message: 'Mismatch\n' +
    '  syntax: auto | <length> | <percentage> | min-content | max-content | fit-content( <length-percentage> )\n' +
    '   value: calc(33.33333% - 10px)!important;\n' +
    '  ------------------------------^',
  rawMessage: 'Mismatch',
  syntax: 'auto | <length> | <percentage> | min-content | max-content | fit-content( <length-percentage> )',
  css: 'calc(33.33333% - 10px)!important;',
  mismatchOffset: 22,
  mismatchLength: 1,
  offset: 22,
  line: 1,
  column: 23,
  loc: {
    source: '<unknown>',
    start: { offset: 22, line: 1, column: 23 },
    end: { offset: 23, line: 1, column: 24 }
  }
}
```

**Comment by lahmatiy** [comment]
Thank you for clarification.
That's not a bug, since `lexer.matchProperty()` test a property name and a value. `!important` is not a part of a value, but a [declaration](https://www.w3.org/TR/css-syntax-3/#declaration-diagram):

![image](https://user-images.githubusercontent.com/270491/113018071-6f0e9480-9180-11eb-9e50-721168e69931.png)

So, you need to discard `!important` somehow before using `matchProperty()`. Another option is to use `matchDeclaration()` e.g.:

```js
const { lexer, parse } = require('css-tree');
console.log(lexer.matchDeclaration(parse('width: calc(33.33333% - 10px)!important', { context: 'declaration' })));
```

**Comment by ArgonAlex** [comment]
Ah ok thanks for the explanation. I might have encountered a bug in `csstree-validator` then, since it's using `matchProperty()` to validate declarations.

**Comment by lahmatiy** [comment]
Will be available in next release today.

**Comment by lahmatiy** [comment]
Fixed in [1.1.3](https://github.com/csstree/csstree/releases/tag/v1.1.3)
